README.md

```diff
@@ -48,16 +48,28 @@ The customers will clone the dev kit using
 They can also simply download the dev kit provided as tar ball from xilinx Alveo Vivado flow [webpage](https://www.xilinx.com/products/boards-and-kits/alveo/u200.html#gettingStarted).
 
 *STEP2*:
+
+setup the software development environment
+
+	source sdk_setup.sh
+	cd sdk
```

Comment:
Just source sdk_setup.sh is sufficient. We do not need to manually do sdk_install.sh. That should be automatically taken care of with source sdk_setup.sh

Reply:
Thanks for the info. This issue has been resolved by the new commit.